wordpress-mobile/gutenberg-mobile
# Fixes insertion of paragraph block at the begging when tapping at the empty space

Fixes #3146

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/29009

## Description
Removes `rootBlockId` from `getBlockCount` since the `blockCount` value is only used in the `addBlockToEndOfPost` function and only for the root block list.

## To test:

1. Create a paragraph with some content
2. Tap on empty space below the paragraph
3. Verify that a new paragraph block is inserted at the end

PR submission checklist:
